tests/test.py

```diff
@@ -1123,6 +1134,27 @@ def run_ansible(*, image: str) -> None:
 
     num_tests_ran += num_successful_tests
 
+    # Test the git clone feature
+    command = "scan_ansible"
+    LOG.debug("Testing scan_ansible against a repository that was cloned at runtime")
+    environment = {}
+    environment["VCS_DOMAIN"] = "github.com"
+    environment["CLONE_REPOSITORIES"] = "seisollc/easy_infra,seisollc/easy_infra"
+    environment["CLONE_PROTOCOL"] = "https"
+
+    # TODO: In the future, migrate this to a general test config
```

Comment:
This is purposefully left in for posterity.  If anyone strongly disagrees we can consider another way to note this